include/symbols.h

```diff
@@ -127,9 +127,6 @@ namespace symbols {
     };
     template<char s> constexpr const char Parenthesis<s>::_symbol[2];
 
-    template class Parenthesis<'('>;
-    template class Parenthesis<')'>;
-
```

Comment:
It wasn't needed to transfer these two lines to `symbols.cpp`, classes are implicitly instantiated when needed at compile time.